supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -35,7 +35,7 @@
 
 [[cd-one]]
 ==== image:images/yes.png[yes] CD #1 (noun)
-*Description*: When referring to a specific CD in the Red Hat Enterprise Linux CD set, it is correct to refer to it as "Red Hat Enterprise Linux CD #1." Avoid using "Red Hat Enterprise Linux CD 1."
+*Description*: When referring to a specific CD in the Red Hat Enterprise Linux CD set, it is correct to refer to it as "Red Hat Enterprise Linux CD #1". Avoid using "Red Hat Enterprise Linux CD 1".
```

Comment:
Another comment on preexisting content here. Instead of 'avoid using', maybe we could be prescriptive and just say: Do not use "Red Hat Enterprise Linux CD 1".

But then, holdup ... CDs? do we need to have an entry for a specific CD in the Red Hat Enterprise Linux CD set still!? 
wdyt @bergerhoffer @inoxx03 ?

Reply:
Thanks so much for your review, @ndeevy -- much appreciated! I agree with your suggested edit (if the entry is retained), but it might be better to open a separate issue to review all uses of "avoid" in the supplementary style guide, in case other occurrences should be similarly updated?

Reply:
@bredamc agreed, I can probably follow up with a PR that addresses the 'avoid's down the line. <thumbs_up> :-]

Reply:
Resolving this comment as per the update on gchat.